src/plugins/saved_objects_management/public/management_section/object_view/components/header.tsx

```diff
@@ -61,6 +61,7 @@ export const Header = ({
       <EuiPageContentHeaderSection>
         <EuiTitle>
           {canEdit ? (
+            // <EuiTitle size="l">
```

Comment:
by mistake, for this I'll replace with EuiText small since its a page content header h1
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/32652829/e704bfff-d271-4e50-898a-e9720b9dc25f)